README.md

```diff
@@ -10,24 +10,20 @@ You need to configure the action in your backend:
 
 ```bash
 # From your Backstage root directory
-yarn add --cwd packages/backend @backstage/plugin-scaffolder-backend-module-copier
+yarn add --cwd packages/backend https://github.com/DiamondLightSource/scaffolder-backend-module-copier.git
```

Comment:
Ideally this would be from NPM. @coretl, how's publishing going?

Reply:
I'm the one that's going to publish it after this gets reviewed and accepted. I was gonna change this after in the npm branch